main.go

```diff
@@ -121,6 +130,50 @@ func main() {
 					os.Exit(1)
 				}
 			}
+		} else if module, _ := tfconfig.LoadModule(dir); len(module.RequiredCore) >= 1 && len(args) == 0 { //if there is a .tfswitchrc file, and no commmand line arguments
+			tfversion := ""
+
+			// we skip duplicated definitions and use only first one
+			tfconstraint := module.RequiredCore[0]
+			tflist, _ := lib.GetTFList(hashiURL, true)
+			fmt.Printf("Reading required version from terraform code, constraint: %s\n", tfconstraint)
+
+			c, err := semver.NewConstraint(tfconstraint)
```

Comment:
`c` is used too far from its declaration thus I suggest a more descriptive name.